src/chunker/layout.js

```diff
@@ -467,233 +530,572 @@ class Layout {
 		}
 
 		if (!node) {
+			// console.log(isElement(container), child(container, offset), temp);
```

Comment:
Can probably remove this or add it for debugging if it's helpful?

Reply:
Will remove.